kernel/src/parquet.rs

```diff
@@ -1,17 +1,13 @@
 //! This module exists to help re-export the version of arrow used by default-engine and other
 //! parts of kernel that need arrow
 
-#[cfg(feature = "arrow_53")]
```

Comment:
I am pretty sure that simply deleting support for the older arrow releases is not the right approach for this crate but it was the most expedient for my purposes